### PR TITLE
Remove unnecessary aliases

### PR DESCRIPTION
**Pull request by vikashvverma**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.
- [x] Improvement
    - [x] Bug fix: **`{IA Name}: Fix 2967`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Remove unnecessary aliases

###### Which issues (if any) does this fix?

Fixes https://github.com/duckduckgo/zeroclickinfo-goodies/issues/2967

###### People to notify (@mention interested parties)

https://duck.co/ia/view/dothraki_cheat_sheet (@varundey) - remove dothraki language
https://duck.co/ia/view/finnish_cheat_sheet (@riqpe) - remove english to finnish
https://duck.co/ia/view/greek_cheat_sheet (@synbit) - remove english to greek
https://duck.co/ia/view/high_valyrian_cheat_sheet (@siddhanjay) - remove game of thrones language
https://duck.co/ia/view/kannada_cheat_sheet (@pnpninja) - remove basic kannada
https://duck.co/ia/view/minionese_cheat_sheet (@swapagarwal) - remove minion language
https://duck.co/ia/view/spanish_cheat_sheet (@rasikapohankar) - remove basic spanish
https://duck.co/ia/view/telugu_cheat_sheet (@roxxup) - remove english to telugu

---

Instant Answer Page: https://duck.co/ia/view/dothraki_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mention
- [ ] @varundey
- [ ] @riqpe
- [ ] @synbit
- [ ] @siddhanjay
- [ ] @pnpninja
- [ ] @swapagarwal
- [x] @rasikapohankar
- [ ] @roxxup

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/finnish.json [hunk 1]
```diff
@@ -2,7 +2,9 @@
     "id": "finnish_cheat_sheet",
     "name": "Finnish Cheat Sheet",
     "description": "Basic Finnish words and phrases",
-    "aliases": ["english to finnish", "basic finnish"],
+    "aliases": [
+        "basic finnish"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/finnish.json]: We don't need this.

File: share/goodie/cheat_sheets/json/high-valyrian.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
     },
 
     "aliases": [
-        "valyrian", "game of thrones language", "valyrian language"
+        "valyrian", "valyrian language"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/high-valyrian.json]: We don't need `valyrian language`.

File: share/goodie/cheat_sheets/json/language/greek.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
        "sourceUrl" : "http://www.omniglot.com/language/phrases/greek.php"
     },
     "aliases": [
-        "english to greek", "basic greek"
+        "basic greek"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/language/greek.json]: We don't need this.

File: share/goodie/cheat_sheets/json/language/spanish.json [hunk 4]
```diff
@@ -9,7 +9,7 @@
     },
 
     "aliases": [
-        "spanish words", "spanish to english", "spanish greetings", "basic spanish", "spanish questions"
+        "spanish words", "spanish to english", "spanish greetings", "spanish questions"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/language/spanish.json]: We don't need `spanish to english`.

File: share/goodie/cheat_sheets/json/language/telugu.json [hunk 5]
```diff
@@ -7,8 +7,7 @@
         "sourceUrl":"http://wikitravel.org/en/Telugu_phrasebook"
     },
     "aliases":[  
-        "english to telugu",
-        "basic telugu"
+        "english to telugu"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/language/telugu.json]: We don't need this.

File: share/goodie/cheat_sheets/json/language/spanish.json (hunk 4 above)
Comment (rasikapohankar) [inline share/goodie/cheat_sheets/json/language/spanish.json]: Hey @GuiltyDolphin I had indented the `trn` fields incorrectly. Is it alright if we fix the indentation here even if it's not related to the issue? Or should I do that in a separate pull request?

Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/language/spanish.json]: @rasikapohankar It's up to you - if you want to fix it go ahead, otherwise sure it can be fixed here.

Comment (vikashvverma) [inline share/goodie/cheat_sheets/json/language/spanish.json]: @rasikapohankar Let me know if you are making a separate PR

Comment (rasikapohankar) [inline share/goodie/cheat_sheets/json/language/spanish.json]: @GuiltyDolphin Thanks @vikashvverma I won't make a separate PR, can you please fix the indentation if its okay with you?

Comment (vikashvverma) [inline share/goodie/cheat_sheets/json/language/spanish.json]: @rasikapohankar I don't see any indentation issue. Can you tell me where it is?

Comment (rasikapohankar) [inline share/goodie/cheat_sheets/json/language/spanish.json]: @vikashvverma At line 25? The `trn` fields are more indented than `key` and `val`. Its not indented more when you see it?

Comment (vikashvverma) [inline share/goodie/cheat_sheets/json/language/spanish.json]: @rasikapohankar My bad! It was not visible in my IDE. Anyway it's formatted now

**Comment by swapagarwal** [comment]
@vikashvverma `minion language` is an alias for `minionese` and both should trigger the cheatsheet.

**Comment by GuiltyDolphin** [comment]
@swapagarwal If you change the alias to `minion` it will still trigger.

**Comment by GuiltyDolphin** [comment]
@vikashvverma I believe this is fully resolved.

File: share/goodie/cheat_sheets/json/language/spanish.json (hunk 4 above)
Comment (rasikapohankar) [inline share/goodie/cheat_sheets/json/language/spanish.json]: Thanks!
